apps/web/src/components/terminal/TerminalHeader.tsx

```
@@ -88,25 +92,30 @@ export function TerminalHeader({
-        className={clsx(
-          'flex items-center gap-2 min-w-0 flex-1',
-          dragHandleProps && 'cursor-grab active:cursor-grabbing'
+      <div className="flex items-center gap-1.5 min-w-0 flex-1">
+        {dragHandleProps && (
+          <div
+            ref={node => dragHandleProps.setNodeRef(node)}
+            {...dragHandleAttributes}
+            {...dragHandleListeners}
+            className="flex items-center cursor-grab active:cursor-grabbing px-0.5 shrink-0 touch-none"
+            aria-label="Drag to reorder"
+          >
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)